src/riak_kv_pb_timeseries.erl

```diff
@@ -152,6 +154,38 @@ process(#tsputreq{table = Table, columns = _Columns, rows = Rows}, State) ->
             {reply, missing_helper_module(Table, BucketProps), State}
     end;
 
+process(#tsputreqttb{rows = []}, State) ->
+    {reply, #tsputresp{}, State};
+process(#tsputreqttb{table = Table, columns = _Columns, rows = Rows}, State) ->
+
+    Mod = riak_ql_ddl:make_module_name(Table),
+    Data = Rows,
+
+    %% validate only the first row as we trust the client to send us
+    %% perfectly uniform data wrt types and order
+    case (catch Mod:validate_obj(hd(Data))) of
```

Comment:
@erikleitch So it looks like we started validating all rows back in https://github.com/basho/riak_kv/pull/1313.  **#tsputreqttb** differs in a few places from **#tsputeq** currently.